src/views/instructor/training/EditSession.vue

```diff
@@ -180,28 +182,42 @@ export default {
 		async submitTraining() {
 			try {
 				// math for get duration of traning session
-				const delta = Math.abs(new Date(this?.session.endTime) - new Date(this?.session.startTime)) / 1000;
+				const delta = Math.abs(new Date(this.session.endTime) - new Date(this.session.startTime)) / 1000;
 				const hours = Math.floor(delta / 3600);
 				const minutes = Math.floor(delta / 60) % 60;
 				this.duration = `${('00' + hours).slice(-2)}:${('00' + minutes).slice(-2)}`;
 
 				// form and sending system traning record to VATUSA API 
-				const formData = new FormData();
-				formData.append('instructor_id', this.session.instructor.cid);
-				formData.append('session_date', this.session.startTime);
-				formData.append('position', this.session.position);
-				formData.append('duration', this.duration);
-				formData.append('movements', this.session.movements);
-				formData.append('score', this.session.progress);
-				formData.append('notes', this.session.studentNotes);
-				formData.append('location', this.session.location);
-				formData.append('ots', this.session.ots);
-				await vatusaApiAuth.post(`/user/${this.session.student.cid}/training/record?test=true`, formData);
+				// const formData = new FormData();
+				// formData.append('instructor_id', this.session.instructor.cid);
+				// formData.append('session_date', this.session.startTime);
+				// formData.append('position', this.session.position);
+				// formData.append('duration', this.duration);
+				// formData.append('movements', this.session.movements);
+				// formData.append('score', this.session.progress);
+				// formData.append('notes', this.session.studentNotes);
+				// formData.append('location', this.session.location);
+				// formData.append('ots', this.session.ots);
+				// await vatusaApiAuth.post(`/user/${this.session.student.cid}/training/record`, formData);
+
+				await vatusaApiAuth.post(`/user/${this.session.student.cid}/training/record` , {
+					"instructor_id": this.session.instructor.cid,
+					"session_date": dayjs(this.session.startTime).format("YYYY-MM-DD HH:mm"),
+					"position": this.session.position,
+					"duration": this.duration,
+					"movements": this.session.movements,
+					"score": this.session.progress,
+					"notes": this.session.studentNotes,
+					"location": this.session.location,
+					"ots": this.session.ots
+				});
+				
 
 				// VATUSA API response
 				this.toastSuccess('Traning Record issued');
 			} catch(e) {
 				this.toastError(e);
+				console.log(e);
```

Comment:
Do we need this now that the feature is done?

Reply:
no, i delete it in new version